src/components/TokenBottomSheet.tsx

```diff
@@ -37,9 +37,12 @@ interface Props<T extends TokenBalance> {
   TokenOptionComponent?: React.ComponentType<TokenOptionProps>
 }
 
-export interface TokenOptionProps {
+export interface TokenBalanceItemOptionProps {
   tokenInfo: TokenBalance
   onPress: () => void
+}
+
+export interface TokenOptionProps extends TokenBalanceItemOptionProps {
```

Comment:
Is this extended prop still used with the changes to the key construction?

Reply:
yean that extended prop is still used in `TokenBottomSheet` which uses the deprecated `TokenOption`. I didn't want to mess with those here since they are not used in my changes and so didn't want to cause more confusion by messing with them.